src/stores/AppStore.tsx

```diff
@@ -67,5 +77,6 @@ export const appStore = AppStore.create({
       view: View.Cell
     }
   },
-  showSubstances: showSubstances
+  showSubstances: showSubstances,
+  availableViews: availableViews
```

Comment:
@sfentress I do really like setting all these values at the start and then just pulling them out of the `store` later